src/private_cuda.h

```diff
@@ -107,7 +107,8 @@ GPUARRAY_LOCAL int cuda_record(gpudata *, int);
 /* private flags are in the upper 16 bits */
 #define CUDA_WAIT_READ  0x10000
 #define CUDA_WAIT_WRITE 0x20000
-#define CUDA_WAIT_MASK  0x30000
+#define CUDA_WAIT_FORCE 0x40000
```

Comment:
I think you should also define GPUARRAY_CUDA_WAIT_FORCE as GPUARRAY_CUDA_WAIT_READ/WRITE. The last 2 are used in Theano.